src/Page/Answers.elm

```diff
@@ -97,12 +117,55 @@ update msg model =
                         |> Task.Extra.execute
                     ]
 
+        PerformCreateMode bool ->
+            { model | creating = bool }
+                |> withNoCmd
+
+        PerformModifyMode maybeId ->
+            case maybeId of
+                Just id ->
+                    case model.state of
+                        Loaded serverAnswers ->
+                            let
+                                answerSingleton =
+                                    List.filter (\ans -> ans.idAnswer == id) serverAnswers
+
+                                probablyAnswer =
+                                    List.head answerSingleton
+
+                                title =
+                                    case probablyAnswer of
+                                        Just answer ->
+                                            answer.title
+
+                                        Nothing ->
+                                            ""
```

Comment:
Maybe something like this might work :

```elm
answer =
    List.filter(\ans -> ans.idAnswer == id) serverAnswers
        |> List.head

title =
    Maybe.map .title answer
        |> Maybe.withDefault ""

desc =
    Maybe.map .description answer
        |> Maybe.withDefault ""
```

which is a bit less verbose than manual case distinction.

Reply:
Noted 👍